assets/templates/carousel-component.html

```diff
@@ -78,14 +79,77 @@ <h3>Carousel Component</h3>
             <hr class="dpsg-break" />
             <div class="dp-container">
               <div class="dp-rowflex">
-                <div class="col-sm-6 m-b-12">
-                  <div class="dp-box-shadow dpsg-content-sample">
-                    <p class="dp-boxshadow">Carousel 1</p>
+                <div class="col-sm-12 col-md-6">
+                  <div class="dp-carousel dp-carousel-orange">
+                    <div class="dp-carousel-wrapper">
+                      <div class="dp-carousel-content" data-pagination="true">
+                        <div class="dp-carousel-slide">
+                          <span>Último BlogPost</span>
+                          <h3>
+                            ¿Cuáles son las diferencias entre servicio al
+                            cliente y atención al cliente?
+                          </h3>
+                          <a
+                            ><span class="ms-icon icon-arrow-next"></span> Ver
+                            más</a
+                          >
+                        </div>
```

Comment:
Inside the slide it would be best to differentiate the slide as an articlie with a title and a description, to make it more semantic, in order to make it easier for accesibility, readers and so on.  Something like this:
```suggestion
                        <article class="dp-carousel-slide">
                          <h3>Último BlogPost</h3>
                          <p>
                            ¿Cuáles son las diferencias entre servicio al
                            cliente y atención al cliente?
                            <a
                            ><span class="ms-icon icon-arrow-next"></span> Ver
                            más</a
                          >
                          </p>
                          
                        </article>
```
This way you would read: a slide has a title, with a description, and the description allows you to know more. 
About [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article)
Do you see it possible?

Reply:
Yes, in the last commit, the title is being an `h3`, the descriptions a `span` and the "ver más" an `a`. Is it ok this way?